WordPress/Resources/AppStoreStrings.po

```diff
@@ -34,17 +34,19 @@ msgid ""
 "View the Privacy Notice for California Users at https://automattic.com/privacy/#california-consumer-privacy-act-ccpa.\n"
 msgstr ""
 
-msgctxt "v19.3-whats-new"
+#. translators: Keywords used in the App Store search engine to find the app.
+#. Delimit with a comma between each keyword. Limit to 100 characters including spaces and commas.
+msgctxt "app_store_keywords"
+msgid "blogger,writing,blogging,web,maker,online,store,business,make,create,write,blogs"
```

Comment:
🆕  Had to re-introduced the entry manually given the bug we currently have in our release toolkit not being able to re-introduce new entries but only being able to copy or update existing ones in the `.po`

Note that I only updated the one for WordPress and not Jetpack, since pdjUbD-8v-p2 only applies to WordPress.